lms/djangoapps/courseware/tests/test_views.py

```diff
@@ -7,6 +7,7 @@
 import itertools
 import json
 import unittest
+import html
```

Comment:
Minor suggestion; would you mind putting this in alphabetical order among the other stdlib imports?  We're not yet running `isort` on each commit to enforce import order, but we do run it occasionally after running refactoring scripts, so this will eventually show up as a change in another PR if it isn't updated now.

Same applies in the other files updated here.

Reply:
Agreed! Thanks for the tip